src/himrepClassifier/classifier.cpp

```diff
@@ -259,7 +259,9 @@ bool Classifier::train(Bottle *locations, Bottle &reply)
     // crop image
     ImageOf<PixelRgb> croppedImg;
     croppedImg.resize(blobW,blobH);
-    toCvMat(*image)(cv::Rect(x_min,y_min,blobW,blobH)).copyTo(toCvMat(croppedImg));
+    for(int j=0 ; j<blobH ; j++)
```

Comment:
We definitely need to resort to OpenCV native functionalities to copy out ROI images and thus avoid relying on for cycles.

Reply:
Ok, I will modify this to avoid for loops.

Is it for code readability or for performance?
Passing though OpenCV to perform cropping seemed a bit inefficient to me because of the channel conversion performed by `toCvMat` and `fromCvMat`.

Reply:
Thanks 👍 
It should be a bit for both. Also, this could represent a reference for cases when in the future we might need to do the same thing, but in the correct manner.

`toCvMat` and `fromCvMat` may or may not spend time performing the operation in question depending on the specific context. Anyway, I think we should start using them quite blindly for switching to/from OpenCV domain and blame @Nicogene for performance issues 😛

Reply:
Ah! I can give help on this from Monday onwards, once back from Milan.

Reply:
In this case, with `PixelRgb`, I think the operation is always performed.

Anyway, I will do the changes tomorrow, in a way that can indeed be reused blindly independently of the context.